networking/synthetic/stac-s2i-builder-image/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM openshift/base-centos7
+
+MAINTAINER Vikas Choudhary "vichoudh@redhat.com"
+
+ENV container docker
+
+ARG github_url_stac_config
+
+RUN git clone ${github_url_stac_config}
+RUN wget http://mirror.centos.org/centos/7/updates/x86_64/Packages/kernel-3.10.0-514.10.2.el7.x86_64.rpm \
```

Comment:
This is going to break :-( Please variabilize like everything else (ansible facts can get you the kernel version).